Description: make anthy.el works with Emacs 24.3.1
 See [Anthy-dev 3852] anthy.el を emacs 24.3.1 と使用できるためのパッチ

Author: Mike FABIAN <maiku.fabian@gmail.com>

---
Origin: other
Forwarded: not-needed
Last-Update: 2013-03-08

--- anthy-9100h.orig/src-util/anthy.el
+++ anthy-9100h/src-util/anthy.el
@@ -892,7 +892,7 @@
 	 ((event-matches-key-specifier-p event 'backspace) 8)
 	 (t
 	  (char-to-int (event-to-character event)))))
-    last-command-char))
+    last-command-event))
 
 ;;
 ;;
